src/components/data-exploration/differential-expression-tool/DiffExprResults.jsx

```diff
@@ -94,7 +95,7 @@ const DiffExprResults = (props) => {
     if (reason === geneTableUpdateReason.loaded || reason === geneTableUpdateReason.loading) {
       return;
     }
-
+    geneTableState.current = newState;
```

Comment:
Very nitpicky thing, but could you call both `loadDifferentialExpression`'s with `geneTableState.current`?